lib/ruby_lsp/tapioca/addon.rb

```diff
@@ -60,6 +60,22 @@ def version
 
       sig { params(changes: T::Array[{ uri: String, type: Integer }]).void }
       def workspace_did_change_watched_files(changes)
+        constants = changes.filter_map do |change|
+          path = change[:uri].gsub("file://", "")
+
+          entries = T.must(@index).entries_for(path, RubyIndexer::Entry::Class, RubyIndexer::Entry::Module)
+          next unless entries
```

Comment:
This shouldn't be needed. If `entries` is `nil`, then it will already not add anything to the result.